metaci/release/utils.py

```diff
@@ -107,6 +122,15 @@ def send_release_webhook(project_config, release, config_item=None):
         json=payload,
         headers={"Authorization": f"Bearer {token}"},
     )
+    release.version_name = release.repo.latest_release.git_tag.strip(
+        f"{release.repo.release_tag_regex}"
+    )
+    release.version_number = release.repo.latest_release.git_tag.strip(
+        f"{release.repo.release_tag_regex}"
+    )
+    release.save()
+    breakpoint()
```

Comment:
Same comment re using release.git_tag instead of release.repo.latest_release.git_tag